tuf/utils/x509.go

```diff
@@ -252,9 +252,31 @@ func ParsePEMPublicKey(pubKeyBytes []byte) (data.PublicKey, error) {
 			return nil, fmt.Errorf("invalid certificate: %v", err)
 		}
 		return CertToKey(cert), nil
+	case "PUBLIC KEY":
+		keyType, err := keyTypeForPublicKey(pemBlock.Bytes)
+		if err != nil {
+			return nil, err
+		}
+		return data.NewPublicKey(keyType, pemBlock.Bytes), nil
 	default:
-		return nil, fmt.Errorf("unsupported PEM block type %q, expected certificate", pemBlock.Type)
+		return nil, fmt.Errorf("unsupported PEM block type %q, expected CERTIFICATE or PUBLIC KEY", pemBlock.Type)
+	}
+}
+
+func keyTypeForPublicKey(pubKeyBytes []byte) (string, error) {
+	pub, err := x509.ParsePKIXPublicKey(pubKeyBytes)
+	if err != nil {
+		return "", fmt.Errorf("unable to parse pem encoded public key: %v", err)
+	}
+	_, ok := pub.(*ecdsa.PublicKey)
+	if ok {
+		return data.ECDSAKey, nil
+	}
+	_, ok = pub.(*rsa.PublicKey)
+	if ok {
+		return data.RSAKey, nil
 	}
+	return "", fmt.Errorf("unkown public key format")
```

Comment:
In `tuf/signed/errors.go` we have `ErrInvalidKeyType` which seems appropriate. It's used to indicate it the key type is invalid/unrecognized in the context of key types Notary accepts, not necessarily that the key is universally invalid.